test/basic.html

```diff
@@ -74,6 +74,35 @@
         }, 12);
       });
 
+      test('show() accepts valid properties', function() {
+        toast = fixture('basic');
+        toast.show({text: 'hello world', duration: 20});
+        assert.isTrue(toast.opened, '`opened` is true');
+        assert.equal(toast.text, 'hello world', '`text` is correct');
+        assert.equal(toast.duration, 20, '`duration` is correct');
+      });
+
+      test('show() does not accept invalid properties', function() {
+        toast = fixture('basic');
+        toast.show({foo: 'bar'});
+        assert.isUndefined(toast.foo, '`foo` is not a valid property and will not be set');
+        assert.isTrue(toast.opened, '`opened` is true');
+      });
+
+      test('show() does not accept private properties', function() {
+        toast = fixture('basic');
+        var temp = toast._manager;
+        toast.show({_manager: 'bar'});
+        assert.notEqual(toast._manager, 'bar', '`_manager` is a private property and will not be set');
```

Comment:
uhm, if you don't want to check for the old value, then you don't need the `temp` variable.
Otherwise you could check if manager was preserved
```javascript
assert.equal(toast._manager, temp, '`_manager` was preserved');
```